### PR TITLE
Fix tenant fetcher to read different field from the event

### PR DESCRIPTION
**Pull request by nyordanoff**
**Description**

This PR changes the field from the event response from which the Tenant Fetcher gets the parent of a subaccount.

Changes proposed in this pull request:
- changed field
- adapted unit and e2e tests

**Pull Request status**

- [x] Implementation
- [x] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/director/internal/tenantfetcher/event_page_test.go [hunk 1]
```diff
@@ -332,7 +333,7 @@ func Test_getTenantMappings(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			events := make([][]byte, 0, len(test.detailsPairs))
 			for i, detailPair := range test.detailsPairs {
-				events = append(events, fixEventWithDetails(fmt.Sprintf("id%d", i), fmt.Sprintf("foo%d", i), "GlobalAccount", constructJSONObject(detailPair...), test.fieldMapping))
+				events = append(events, fixEventWithDetailsWithoutGA(fmt.Sprintf("id%d", i), fmt.Sprintf("foo%d", i), "GlobalAccount", constructJSONObject(detailPair...), test.fieldMapping))
```
Comment (DimitarPetrov) [inline components/director/internal/tenantfetcher/event_page_test.go:336]: The differentiation between GA events and SA events can be removed here as well.

File: tests/tenant-fetcher-job/tests/handler_test.go [hunk 2]
```diff
@@ -75,11 +75,11 @@ const (
 		"guid": "%s",
 		"displayName": "%s",
 		"subdomain": "%s",
-		"parentGuid": "%s",
```
Comment (DimitarPetrov) [inline tests/tenant-fetcher-job/tests/handler_test.go:78]: Leave it as well with a different value from the globalAccountGUID field to assert that the feature actually works.

**Review by DimitarPetrov — CHANGES_REQUESTED** [review]

File: components/director/internal/tenantfetcher/event_page_test.go (hunk 1 above)
Comment (nyordanoff) [reply components/director/internal/tenantfetcher/event_page_test.go:336]: Yes, I just adapted it after discussing the other occurrence in the service tests.

**Review by DimitarPetrov — APPROVED** [review]

**Pull request merged**